src/lasp_core.erl

```diff
@@ -190,6 +191,32 @@ read(Id, Threshold, Store) ->
             end,
     read(Id, Threshold, Store, Self, ReplyFun, BlockingFun).
 
+%% @doc Perform a delta read for a particular identifier.
+%%
+%%      Given an `Id', perform a blocking read until a delta is
+%%      delivered.
+%%
+%%      This operation blocks until any deltas have been delivered.
+%%
+-spec read_delta(id(), value(), store()) -> {ok, {delta, var()}}.
+read_delta(Id, Threshold, Store) ->
+    case Threshold of
+        {strict, undefined} ->
+            %% The first run uses the current value.
+            lasp:read(Id, undefined);
+        {strict, _} ->
+            %% From the second run, it always waits for a delta
+            %% (from the local or the remote).
+            Self = self(),
+            {ok, Object} = do(get, [Store, Id]),
+            WDT = lists:append(Object#dv.waiting_delta_threads, [{delta, Self}]),
+            do(put, [Store, Id, Object#dv{waiting_delta_threads=WDT}]),
+            receive
```

Comment:
Eventually, this should be parameterized as the other ```read``` operation is.  Maybe leave a ```@todo``` here so we remember?